docs/assets/funfacts.txt

```diff
@@ -10,22 +11,23 @@ At one point, Feather Client was willing to pay $300 USD to add Skytils.
 SkyblockPersonalized was once the cause of many rickrolls from right-clicking\nany of the Mathematics Farming Hoes. (It's been removed by now, of course.)
 The mod NEU has over 80k lines of code, making it\none of the largest mods out there for Skyblock.
 The icon you see in SkyClient's main menu and other places\nis made by none other than the esteemed Zetvue#9664.
-Most of SkyClient's staff have not touched Hypixel Skyblock for several months.
+Most of SkyClient's staff have not touched Hypixel Skyblock for several months. Quit today!
```

Comment:
does this go off screen

Reply:
Try it and see!

Reply:
If it does just remove it

Reply:
It shouldn't as other lines are bigger then it (that aren't split with \n)